### PR TITLE
Docs: Add query caching to enterprise docs page

### PR DESCRIPTION
**Pull request by mjseaman**
**What this PR does / why we need it**:
Add a line for Query caching to the main Enterprise docs page, along with the other Enterprise features.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by joanlopez — APPROVED** [review]
LGTM!

File: docs/sources/enterprise/_index.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ With Grafana Enterprise, you get access to new features, including:
 
 - [Fine-grained access control]({{< relref "access-control/_index.md" >}}) to control access with fine-grained roles and permissions.
 - [Data source permissions]({{< relref "datasource_permissions.md" >}}) to restrict query access to specific teams and users.
+- [Data source query caching]({{< relref "query-caching.md" >}}) to temporarily store query results in Grafana, reducing data source load and rate limiting.
```
Comment (osg-grafana) [inline docs/sources/enterprise/_index.md:48]:
```suggestion
- [Data source query caching]({{< relref "query-caching.md" >}}) to temporarily store query results in Grafana to reduce data source load and rate limiting.
```

**Comment by joanlopez** [comment]
It should be backported to `v8.0.x` and `v7.5.x` release branches as well, right? 🤔

**Comment by mjseaman** [comment]
> It should be backported to `v8.0.x` and `v7.5.x` release branches as well, right? 🤔

@joanlopez yes.. How do I do that?

**Comment by hugohaggmark** [comment]
Hi, not sure we need to backport docs? Can you verify this @grafana/docs-squad?

**Comment by osg-grafana** [comment]
@mjseaman, there are some security vulnerabilities associated with this PR. Please have a look.

**Comment by osg-grafana** [comment]
There needs to be two issues (one per milestone) from what I understand. @oddlittlebird and @achatterjee-grafana, is "Backporting" another term for cherry picking into a particular release?

**Comment by osg-grafana** [comment]
@mjseaman, why was this merged with those build issues?

**Comment by mjseaman** [comment]
This PR was a one-line docs update. The security vulnerabilities were from other unchanged files, with a new Github beta feature that identifies vulnerabilities with every PR.

**Comment by mjseaman** [comment]
Screen Shot 2021-06-01 at 10 39 26 AM

**Pull request merged**